backend/app/services/adapters/executor_kinds.py

```
@@ -1218,6 +1219,38 @@ def _format_subtasks_response(
+            system_mcp_config = None
+            if is_subscription and task_token:
+                try:
+                    from app.mcp_server.server import get_mcp_system_config
+
+                    # Get backend URL from settings
+                    backend_url = settings.BACKEND_INTERNAL_URL
+                    system_mcp_config = get_mcp_system_config(backend_url, task_token)
+                    logger.info(
+                        f"Generated system MCP config for subscription task {subtask.task_id}"
+                    )
+                except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)